src/frontend/src/app/audit-log/audit-log.component.ts

```diff
@@ -91,7 +164,11 @@ export class AuditLogComponent implements OnInit {
   }
 
   auditLogSubscriber() {
-    this.auditService.getAuditLogs(this.limit, this.offset, this.source, this.severity).
+    console.log('Limit: ', this.limit);
+    console.log('offset: ', this.offset);
+    console.log('tempOffset: ', this.tempOffset);
+    console.log('recordCount: ', this.recordCount);
+    this.auditService.getAuditLogs(this.limit, this.tempOffset, this.source, this.severity).
```

Comment:
make above log one (or max 2 liner) e.g. `console.log('Limit: ', this.limit, 'Offset: ', this.offset);`